src/test/java/com/jwplayer/southpaw/MockState.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
I can see this being moved into `southpaw/state/InMemoryState` in the future, but left this as a `MockState` for now as it only implements enough for what existing tests would need to mock